app/views/layouts/application.html.haml

```diff
@@ -5,6 +5,11 @@
     %meta{:name => 'viewport', :content => 'width=device-width, initial-scale=1'}/
     %meta{'http-equiv' => 'X-UA-Compatible', :content => 'IE=edge'}/
 
+    %link{:rel => "apple-touch-icon", :sizes => "180x180", :href => "/assets/apple-touch-icon.png"}/
+    %link{:rel => "manifest", :href => "/assets/manifest.json"}/
```

Comment:
Oh, yes. You're right. @Gargron any preferences on where to put those two files?